test/e2e/framework/deployment.go

```diff
@@ -43,12 +43,28 @@ const HTTPBunService = "httpbun"
 // NipService name of external service using nip.io
 const NIPService = "external-nip"
 
+// HTTPBunImage is the default image that is used to deploy HTTPBun with the framwork
+const HTTPBunImage = "registry.k8s.io/ingress-nginx/e2e-test-httpbun:v20230505-v0.0.1"
```

Comment:
Can we get this info from an environment variable? So we can default in Makefile and not need to generate an e2e runner every time we change it